src/cclib/parser/qchemparser.py

```diff
@@ -804,6 +808,26 @@ def extract(self, inputfile, line):
                     polarizability.append(line.split())
                 self.polarizabilities.append(numpy.array(polarizability))
 
+            # Static polarizability from finite difference.
+            if line.strip() == 'Static polarizability tensor [a.u.]':
+                if not hasattr(self, 'polarizabilities'):
+                    self.polarizabilities = []
+                polarizability = []
+                for _ in range(3):
+                    line = next(inputfile)
+                    polarizability.append(line.split())
```

Comment:
Consider using comprehension here:
polarizability = [next(inputfile).split() for _ in range(3)]